src/controller/python/test/test_scripts/base.py

```diff
@@ -356,6 +358,38 @@ def TestFailsafe(self, nodeid: int):
             return True
         return False
 
+    async def TestCaseEviction(self, nodeid: int):
+        self.logger.info("Testing CASE eviction")
+
+        minimumCASESessionsPerFabric = 3
+        minimumSupportedFabrics = 16
+
+        #
+        # This test exercises the ability to allocate more sessions that is supported in the
+        # pool configuration. By going beyond (minimumSupportedFabrics * minimumCASESessionsPerFabric),
+        # it starts to test out the eviction logic. This specific test does not validate the specifics
+        # of eviction, just that allocation and CASE session establishment proceeds successfully on both
+        # the controller and target.
+        #
+        for x in range(minimumSupportedFabrics * minimumCASESessionsPerFabric * 3):
```

Comment:
What is the `*3` magic number?

Reply:
As per the previous comment, it attempts to go beyond (minimumSupportedFabrics * minimumCASESessionsPerFabric) by some arbitrary number.